maintainers/working_groups/learning_journey/tutorial-workflow.md

```diff
@@ -0,0 +1,81 @@
+# Tutorial Workflow
+
+This document describes the recommended workflow when creating tutorials for the learning journey tutorial series.
+
+## Writing process
+### Pick a tutorial
+There are [tracking issues][tracking_issues] for tutorials that the working group has decided should exist as part of the tutorial series.
+Pick an issue that covers a topic that you're either knowledgeable about or have a particular interest in, then comment on the issue indicating your interest in working on the tutorial.
+
+The tracking issue will contain status information such as whether PRs have been submitted to create an outline or a draft of the article or whether someone else has already started work on the tutorial.
+Make sure to read the issue fully to ensure that you don't duplicate work that someone else is already doing!
+
+[tracking_issues]: https://github.com/NixOS/nix.dev/issues?q=is%3Aissue+is%3Aopen+label%3Atracking+label%3A%22learning+journey%22
+
+### Submit an outline PR
+Work on the tutorial series will take place in the `learning-journey` branch.
+Submit a PR with an outline of the tutorial using the [tutorial outline template][tutorial_outline_template], making sure to submit the PR against the `learning-journey` branch.
+The outline should be placed in the [tutorial outlines directory][outlines_dir].
+The outline will go through a review process to ensure that the material is satisfactory and is detailed enough that any other contributor could pick it up and work on it.
+In particular, the tutorial outline should:
+
+- Be a tutorial as defined in the [Diataxis][diataxis_tutorial] framework.
+- Clearly state the prerequisite knowledge.
+- Have a well-defined scope and learning objective.
+
+[tutorial_outline_template]: ./tutorial-outlines/template.md
+[outlines_dir]: ./tutorial-outlines/
+[diataxis]: https://diataxis.fr/tutorials/
+
+### Submit a draft PR
+Once the outline PR has been merged, work can begin on a draft of the tutorial.
+Submit a PR with a draft of the tutorial using the [tutorial template][tutorial_template], placing the draft in the `source/tutorials/learning-journey`.
+The PR should be opened against the `learning-journey` branch.
```

Comment:
- What's the process for merging the learning-journey branch back into master? Branch otherwise diverges from master and needs maintenance
- What's the purpose of having a separate branch entirely?
  - Clean history on master maybe? Doesn't seem that important
- Alternative: Separate open PR's open for each tutorial, as we already do
- For just committing outlines, we don't need a separate branch, a separate directory as described here is enough

Reply:
Note: We did this togother in todays documentation team meeting

Reply:
I wrote it this way because we talked about following a similar workflow that the documentation survey is using. I don't actually have a strong preference one way or the other. The main issue I see is that (as far as I'm aware) we currently don't have a way to mark documents as drafts and have Sphinx ignore them. I think if you include the outlines next to other published pages you'll get errors from Sphinx at build time complaining that you have documents not included in the site. Maybe @yukiisbored can chime in on this since they're the resident Sphinx wizard.